packages/example/src/pages/components/MiniCard.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: MiniCard
+description: Usage instructions for the MiniCard component
+---
+
```

Comment:
@jeanservaas can you provide some context on when someone would reach for this card as opposed to a resource card? It'd be great to have some design intent rather than just having folks choose. Some sites/projects might not have a designer to pick between the two.

Reply:
> Dope! Coming together. If I remember correctly, we nuked the `Children.map` stuff and settled on moving the column into the MiniCard component (ala DoDont).
> 
> That way each card could get their own column props rather than forcing them all to use the same (like it would be if we applied them with Children.Map)

Yeah, decided to keep the Children.map and keep the Column with the CardGroup because when i remove the Children.map and move the Column to the MiniCard itself it started to break down the aside case that Jeannie was looking for.